### PR TITLE
Update versions for EH 5.2.0 release

### PR DESCRIPTION
**Pull request by srnagar**
This PR is to update versions of Event Hubs and Checkpoint Store libraries to prepare for GA release.

### CODE REVIEW

File: sdk/eventhubs/azure-messaging-eventhubs/README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ platform. First, add the package:
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity</artifactId>
-    <version>1.0.3</version>
+    <version>1.1.2</version>
```
Comment (samvaity) [inline sdk/eventhubs/azure-messaging-eventhubs/README.md:103]: Unrelated to this PR, but should we remove this from the readme. I don't think thats the pattern we follow for other SDK's readme to have identity version info in the readme.

**Review by samvaity — APPROVED** [review]

**Comment by srnagar** [comment]
@alzimmermsft I learned that Storage Blobs is releasing OOB this month and I don't think we need to wait on storage release for this cycle. @samvaity can you please confirm?

**Comment by samvaity** [comment]
>Should this wait for Storage releases?

@alzimmermsft Storage blob isn't releasing this cycle and I am guessing EH already has a stable dependency on blob so we should be good to proceed for EH releases/

File: sdk/eventhubs/azure-messaging-eventhubs/README.md (hunk 1 above)
Comment (srnagar) [reply sdk/eventhubs/azure-messaging-eventhubs/README.md:103]: I see that all services have this in readme. It's required in the "Authenticate the client" section.

Comment (samvaity) [reply sdk/eventhubs/azure-messaging-eventhubs/README.md:103]: Oh yes! I was confused if we actually kept it or not, you are good!

**Pull request merged**